Moesif.Api/Controllers/BaseController.cs

```diff
@@ -51,7 +51,7 @@ public static IHttpClient ClientInstance
         internal void ValidateResponse(HttpResponse _response, HttpContext _context)
         {
             if ((_response.StatusCode < 200) || (_response.StatusCode > 206)) //[200,206] = HTTP OK
-                throw new APIException(@"HTTP Response Not OK", _context);
+                throw new APIException($"HTTP Error-response: [status: {_response.StatusCode}, error:{_response.RawBody}]"));
```

Comment:
Maybe keep starting of string to "HTTP Response Not OK" in case anyone using the SDK is checking for existence of this string? 
Maybe something like:
```
"HTTP Response Not OK [status: {_response.StatusCode}, error:{_response.RawBody}]"
```